rubric/rubric_quality.md

```diff
@@ -12,9 +12,9 @@
   <tr>
     <td>Did not attempt</td>
     <td>Code is difficult to read and understand due to many issues that affects readability. Code is also poorly organized.<br></br> No effort has been made to reduce code repetition. Tests are absent.</td>
-    <td>Code is generally easy to read and understand with few non-reoccurring issues and at most two reoccurring issue that affects readability. <br></br> Multiple issues with code repetition exist, and several tests are absent and/or are of poor efficacy</td>
+    <td>Code is fairly readable but not very easy to understand. Sufficient and proper commenting/documentation is missing. A few non-reoccurring, and at most two reoccurring issues exist that affect readability.<br></br> Multiple issues with code repetition exist, and several tests are absent and/or are of poor efficacy</td>
     <td>Code is generally easy to read and understand with few non-reoccurring issues and at most one reoccurring issue that affects readability. <br></br> Some form of re-occuring code repetition exists, or tests efficacy is poor. </td>
     <td>Code is easy to read and understand with only 1-2 minor and non-reoccurring issues that affect readability. <br></br> Code repetition is mostly minimized and effective tests are present for most functions.</td>
-    <td>Code is exceptionally easy to read and understand.<br></br> For example, variable names are clear, an appropriate amount of whitespace is used to maximize visibility, tabs and spaces are not mixed for indentation, sufficient comments are given.<br></br> All functions have proper documentation (e.g. docstrings in Python). <br></br> Overall, the code is extremely well organized and documented. <br></br>Code repetition is minimized via the use of loops/mapping functions, functions or classes or scripts/files as needed without becoming overly complicated. <br></br> Functions are short, concise, and cohesive without losing clarity; code can be easily modified. <br></br> Tests are present to ensure functions work as expected. Exceptions are caught and thrown if necessary (Once students have learned about exceptions).</td>
```

Comment:
Do we really need the adverbs "exceptionally" and "extremely" in the following sentences, respectively? I would get rid of it. (Seems like I'm not a big fan of adverbs 😉.)

> Code is exceptionally easy to read and understand.

> Overall, the code is extremely well organized and documented.